external/js/kyber/package.json

```diff
@@ -1,16 +1,17 @@
 {
-  "name": "@dedis/kyber-js",
+  "name": "@dedis/kyber",
   "version": "0.0.10",
```

Comment:
since the name is changing, please reset the version to 0.0.1. We will set it to 3.0.0 soon enough anyway.